py/visdom/__init__.py

```diff
@@ -2128,6 +2128,42 @@ def stem(self, X, Y=None, win=None, env=None, opts=None):
 
         return self.scatter(X=data, Y=labels, opts=opts, win=win, env=env)
 
+
+    @pytorch_wrap
+    def sunburst(self, X,parents,labels, win=None, env=None, opts=None):
+        opts = {} if opts is None else opts
+        _title2str(opts)
+        _assert_opts(opts)
+        
+        font_size=opts.get("size")
+        font_color=opts.get("font_color")
+        opacity=opts.get("opacity")
+        line_width=opts.get("marker_width")
+
+        X = np.squeeze(X)
+        assert X.ndim == 1, 'X should be one-dimensional'
```

Comment:
Outside of this, reasonable assertions would be that `labels` and `parents` are the same size, and another for that `values` is as well (if `values` is provided).